test/test.mjs

```diff
@@ -1056,9 +1056,6 @@ async function closeSession(browser) {
       continue;
     }
     if (session.browser !== undefined) {
-      for (const page of await session.browser.pages()) {
-        await page.close();
-      }
       await session.browser.close();
```

Comment:
@orkon this actually fixes an issue with `browsingContext.close()` in puppeteer with Firefox when the `context` as passed into the BiDi call is `null`. Could this be a bug in Puppeteer? Or will some internal state change for other tabs as well when trying to close them all via such a loop?